File: FirstApp.WebAPI/Entities/Employee.cs
using System.ComponentModel.DataAnnotations;

namespace FirstApp.WebAPI
{
    public class Employee
    {
        [Key]
        public string EmpId { get; set; } = Guid.NewGuid().ToString();
        public required string DisplayName { get; set; }
        public required string Email { get; set; }
        public required byte[] PasswordHash { get; set; }
        public required byte[] PasswordSalt { get; set; }
        public bool IsActive { get; set; } = true;
    }
}

File: client/src/types/employee.ts
export type Employee = {
    empId: number;
    displayName: string;
    email: string;
    // token: string;
    imageUrl?: string;
}
export type LoginCreds = {
    email: string;
    password: string;
}
export type RegisterCreds = {
    displayName: string;
    email: string;
    password : string;
}